build/source/engine/coupled_em.f90

```diff
@@ -129,11 +178,13 @@ lightSnow            ! maximum interception capacity an inverse function of new
  real(dp)                             :: dt_frac                ! fraction of time step (-)
  integer(i4b)                         :: nTemp                  ! number of temporary sub-steps
  integer(i4b)                         :: nTrial                 ! number of trial sub-steps
+ logical(lgt)                         :: firstStep              ! flag to denote if the first time step
  logical(lgt)                         :: rejectedStep           ! flag to denote if the sub-step is rejected (convergence problem, etc.)
- logical(lgt),parameter               :: checkTimeStepping=.false.  ! flag to denote a desire to check the time stepping 
+ logical(lgt),parameter               :: checkTimeStepping=.false.      ! flag to denote a desire to check the time stepping 
+ logical(lgt),parameter               :: backwardsCompatibility=.true.  ! flag to denote a desire to ensure backwards compatibility 
```

Comment:
with a previous branch, so that we can check output matches to within machine precision.